test/test_quantized.py

```diff
@@ -593,6 +593,39 @@ def test_cat(self, X, num, dim, relu):
             cat_q = q_cat_op(tensors_q, dim=ch_axis, scale=scale,
                              zero_point=zero_point)
 
+    """Tests quantize concatenation (both fused and not)."""
+    @no_deadline
+    @given(X=hu.tensor(shapes=hu.array_shapes(min_dims=4, max_dims=4,
+                                              min_side=1, max_side=10),
+                       qparams=hu.qparams()),
+           relu=st.booleans(),
+           out_qparams=hu.qparams())
+    def test_cat_nhwc(self, X, relu, out_qparams):
+        X, (scale, zero_point, torch_type) = X
+        out_scale, out_zero_pt, out_torch_type = out_qparams
+        assume(out_torch_type == torch_type)
+
+        # Tile out X so # channels is > 64
+        X = np.repeat(X, 70 / X.shape[3], 3)
+        X = torch.from_numpy(np.ascontiguousarray(X))
+        Y = torch.neg(X)
+        qX = torch.quantize_linear(X, scale, zero_point, torch_type).permute([0, 3, 1, 2])
```

Comment:
I'm not 100% sure about intend here, but `permute([0, 3, 1, 2])` will not give you proper tensor, it will become NHWC in terms of dimension order. When you likely need NCHW dimensions and NHWC strides.

Reply:
No, I want it to be laid out as NHWC in memory and with NCHW strides

Reply:
If `torch.quantize_linear(X, scale, zero_point, torch_type)` returns NHWC dims and NHWC strides you can resolve this conversion

Reply:
Yeah, that's the case. Currently `quantize_linear` returns a fully contiguous tensor